packages/paste-website/src/pages/components/radio-button-group/index.mdx

```diff
@@ -0,0 +1,250 @@
+
+## Examples
+
+### Text
+
+Use the text variant with short labels, no more than 3 words per label.
+
+<LivePreview scope={{RadioButtonGroup, RadioButton}} language="jsx" noInline>
+  {TextExample}
+</LivePreview>
+
+### Text + Icon
+
+Use the text + icon variant when icons are not easily recognizable (e.g., product-specific icons) and a text label would help with understanding. If an icon has a chance of not being completely understood at a glance, pairing it with a text label is recommended.
+
+<LivePreview scope={{RadioButtonGroup, RadioButton, CallIcon, SMSCapableIcon, EmailIcon}} language="jsx" noInline>
+  {TextAndIconExample}
+</LivePreview>
+
+## States
+
+### Error
+
+<LivePreview scope={{RadioButtonGroup, RadioButton}} language="jsx" noInline>
+  {ErrorExample}
+</LivePreview>
+
+### Disabled
+
+<LivePreview scope={{RadioButtonGroup, RadioButton}} language="jsx" noInline>
+  {DisabledExample}
+</LivePreview>
+
+## Composition notes
+
+Radio Button Groups:
+
+- Should have between 2–6 options
+- Are comprised of mutually exclusive options
+- Have a default option selected when possible
+- When using the icon-only variant, pair it with a tooltip
+- Labels should use sentence case
+
+## When to use Radio Button Group
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use Radio Button Group when options are mutually exclusive and only one option is valid."
+    preview={false}
+  />
+  <Dont title="Don't" body="Don’t use Radio Button Groups if you need to select multiple options." preview={false} />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Group related buttons in a Radio Button Group (e.g., types of graphs)." preview={false} />
+  <Dont title="Don't" body="Don’t group buttons together that are not related to each other." preview={false} />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Pair icon-only variants with a tooltip." preview={false} />
+  <Dont
+    title="Don't"
+    body="Don’t use the icon-only variant without a tooltip if the icons are not easily recognizable."
+    preview={false}
+  />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use Radio Button Group to let users select one option from a list." preview={false} />
+  <Dont
+    title="Don't"
+    body="Don’t use Radio Button Groups for actions, such as save, edit, or delete. Use single Buttons instead."
+    preview={false}
+  />
+</DoDont>
+
+## Implementation Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/radio-button-group - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {RadioButtonGroup, RadioButton} from '@twilio-paste/core/radio-button-group';
+
+const Component = () => (
+  <RadioButtonGroup attached>
+    <RadioButton>One</RadioButton>
+    <RadioButton>Two</RadioButton>
+    <RadioButton>Three</RadioButton>
+  </RadioButtonGroup>
+);
+```
+
+#### Radio Button Group Props
+
+All the valid HTML attributes for `div` are supported including the following props:
```

Comment:
issue: we're missing some props here, and the `div` here should be `input[type=radio]` the way we say on the [radio docs](https://paste.twilio.design/components/radio-group/#radio-props). The props id, value, name, disabled, and onChange are missing from RadioButton, and RadioButtonGroup